src/qml/filters/deband/ui.qml

```diff
@@ -0,0 +1,430 @@
+            // Custom preset
+            filter.set(thr1Param, thr1Default)
+            filter.set(thr2Param, thr2Default)
+            filter.set(thr3Param, thr3Default)
+            filter.set(thr4Param, thr4Default)
+            filter.set(linkParam, linkDefault)
+            filter.set(rangeParam, rangeDefault)
+            filter.set(directionParam, directionDefault)
+            filter.set(blurParam, blurDefault)
+            filter.set(couplingParam, couplingDefault)
+            filter.savePreset(allParams, qsTr('Average strength'))
+
+            // Custom preset
+            filter.set(thr1Param, 0.0150291)
+            filter.set(thr2Param, 0.03994)
+            filter.set(thr3Param, 0.0150291)
+            filter.set(thr4Param, 0.0150291)
+            filter.set(linkParam, false)
+            filter.set(rangeParam, 90)
+            filter.set(directionParam, directionMax)
+            filter.set(blurParam, true)
+            filter.set(couplingParam, false)
+            filter.savePreset(allParams, qsTr('Blue sky'))
+
+            // Custom preset
+            filter.set(thr1Param, 0.0150291)
+            filter.set(thr2Param, 0.0150291)
+            filter.set(thr3Param, 0.03994)
+            filter.set(thr4Param, 0.0150291)
+            filter.set(linkParam, false)
+            filter.set(rangeParam, 44)
+            filter.set(directionParam, directionMax)
+            filter.set(blurParam, true)
+            filter.set(couplingParam, false)
+            filter.savePreset(allParams, qsTr('Red sky'))
+
+            // Custom preset
+            filter.set(thr1Param, 0.0100294)
+            filter.set(thr2Param, 0.0100294)
+            filter.set(thr3Param, 0.0100294)
+            filter.set(thr4Param, 0.0100294)
+            filter.set(linkParam, true)
+            filter.set(rangeParam, 23)
+            filter.set(directionParam, directionMax)
+            filter.set(blurParam, true)
+            filter.set(couplingParam, false)
+            filter.savePreset(allParams, qsTr('Minimal strength'))
+
+            // Custom preset
+            // Same as "Minimal" preset for now
+            filter.savePreset(allParams, qsTr('Full range to limited range'))
+
+            // Default preset
+            // Same as "Full to limited" preset; assumed most common use case
+            filter.savePreset(allParams)
+        }
+        filter.blockSignals = false
+
+        setControls()
+    }
+
+
+    width: 500
+    height: 360
+
+
+    GridLayout {
+        columns: 3
+        anchors.fill: parent
+        anchors.margins: 8
+
+        // Row split
+
+        Label {
+            text: qsTr('Preset')
+            Layout.alignment: Qt.AlignRight
+        }
+        Shotcut.Preset {
+            id: idPreset
+            Layout.columnSpan: 2
+            parameters: allParams
+            onPresetSelected: setControls()
+        }
+
+        // Row split
+
+        Label {
+            text: qsTr('Contrast Threshold')
```

Comment:
Per our [UI Conventions](https://shotcut.org/notes/ui-conventions/) field labels should not capitalize each word. I realize there is some inconsistency here, but I want to avoid it continuing, and this is a simple fix. The reason existing ones have not been corrected when I noticed is because I did not want to break translations for something superficial.